lib/std/math/nextafter.zig

```diff
@@ -144,7 +144,7 @@ test "math.nextAfter.int" {
 }
 
 test "math.nextAfter.float" {
-    @setEvalBranchQuota(2000);
+    @setEvalBranchQuota(3000);
```

Comment:
For some reason the changes in `Sema` made this test hit the old quota of 2000. Perhaps someone with better knowledge of the compiler internals could help me understand why this happened?

Reply:
Actually this makes perfect sense, `floatTrueMin` and `floatMax` are inline functions so those functions are now evaluated at comptime.